homework-submission/mobx-week1/src/Componants/TodoList.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { inject, observer } from "mobx-react";
+
+@inject("TodoStore")
+@observer
+class TodoList extends React.Component {
+  state = {
+    clicked: false
+  };
+
+  onEdit = () => {
+    this.setState({
+      clicked: true
+    });
+  };
+  onCancel = () => {
```

Comment:
Oh! you have here `onCanel` and before it line 11 `onEdit`, also you have on line 27 a `setState` for setting the state to be as cancel.

I think we should avoid repeating code for a bit and may be do the following?
```javascript
  toggleEdit = () => {
    this.setState({
      clicked: !this.state.clicked
    });
  }
```

Now this method can be used every where you have edit event or cancel or after submit 🎉